.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
 
-* @ONSdigital/dissemination-platform
+* @ONSdigital/dissemination-archivists
```

Comment:
Does this team need to be created in github to make this file valid?

Reply:
It already exisits 👍 
https://github.com/orgs/ONSdigital/teams/dissemination-archivists

Reply:
I see an error saying "Unknown owner: make sure the team @ONSdigital/dissemination-archivists exists, is publicly visible, and has write access to the repository".

Reply:
Hmmm I hoping its just a github oddity, it seesm to work here fine 🧐 
https://github.com/ONSdigital/dp-apipoc-server/blob/develop/.github/CODEOWNERS